jbrowse/src/client/JBrowse/Browser/plugins/ExtendedVariantPlugin/ExtendedVariantDisplay/model.js

```diff
@@ -25,16 +25,16 @@ export default jbrowse => {
 
     .actions(self => ({
       selectFeature(feature) {
-        const extendedVariantDisplayConfig = getContainingTrack(self).configuration.metadata.value.extendedVariantDisplayConfig
         const trackId = getContainingTrack(self).configuration.trackId
         const session = getSession(self)
+        const metadata = getContainingTrack(self).configuration.metadata.value
         var widgetId = 'Variant-' + trackId;
 
         const featureWidget = session.addWidget(
           'ExtendedVariantWidget',
           widgetId,
           { featureData: feature.toJSON(),
-            extendedVariantDisplayConfig: extendedVariantDisplayConfig },
```

Comment:
same as above. i dont think we should take metadata verbatim

Reply:
see above